src/Lecture2.hs

```diff
@@ -16,6 +16,7 @@ Unlike exercises to Lecture 1, this module also contains more
 challenging exercises. You don't need to solve them to finish the
 course but you can if you like challenges :)
 -}
+{-# OPTIONS_GHC -Wno-incomplete-patterns #-}
```

Comment:
Ideally, you shouldn't disable this warning and should fix all such warnings because they can potentially lead to having partial functions ⚠️